src/WebJobs.Extensions.DurableTask/Correlation/TelemetryActivator.cs

```diff
@@ -81,14 +81,12 @@ public void Dispose()
         public void Initialize(ILogger logger)
         {
             this.endToEndTraceHelper = new EndToEndTraceHelper(logger, this.options.Tracing.TraceReplayEvents);
-            this.endToEndTraceHelper.ExtensionWarningAnnouncement(
-                "Durable Functions Distributed Tracing V2 is GA now! Fore more information, please visit, "
-                + "https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-diagnostics?tabs=csharp-inproc#distributed-tracing.");
 
             if (this.options.Tracing.DistributedTracingEnabled)
             {
                 if (this.options.Tracing.Version == Options.DurableDistributedTracingVersion.None)
                 {
+                    this.EmitDTV2Announcement();
```

Comment:
I was actually thinking that we'd *not* emit the warning in this case. Basically, the customer has explicitly opted not to use distributed tracing v2, so we don't need to pester them with warnings. The warning is really only for customers that aren't aware of distributed tracing v2 - so we'd target folks that don't have anything enabled (covered below in your final `else` clause) or those who are running V1.